docs/website/docs/dlt-ecosystem/verified-sources/airtable.md

```diff
@@ -0,0 +1,230 @@
+---
+title: Airtable
+description: dlt verified source for Airtable
+keywords: [airtable api, airtable verified source, airtable]
+---
+
+# Airtable
+
+Airtable is a cloud-based platform that merges spreadsheet and database functionalities for easy
+data management and collaboration.
+
+This Airtable `dlt` verified source and
+[pipeline example](https://github.com/dlt-hub/verified-sources/blob/master/sources/airtable_pipeline.py)
+loads data using “Airtable API” to the destination of your choice.
+
+Sources and resources that can be loaded using this verified source are:
+
+| Name              | Description                                |
+| ----------------- | ------------------------------------------ |
+| airtable_source   | Retrieves tables from an airtable base     |
+| airtable_resource | Retrives data from a single airtable table |
+
+## Setup Guide
+
+### Grab Airtable personal access tokens
```

Comment:
I'd prefer more formal language. But, that's a matter of taste. If you change it, make sure to also update the link to this page in the other PR: https://github.com/dlt-hub/verified-sources/pull/263/files
```suggestion
### Obtain Airtable personal access tokens
```

Reply:
we have the same header in all our verified sources documents, if we change it here we will have to change it everywhere, maybe next time

Reply:
Makes sense, let's keep it then!

Reply:
i like obtain better too

Reply:
Let it be Grab, as we use it in all pipeline docs. Please!